kafka-client/src/main/java/dev/responsive/kafka/internal/db/LwtWriterFactory.java

```diff
@@ -85,6 +85,12 @@ public RemoteWriteResult<P> setOffset(final long offset) {
     builder.addStatement(fencingStatement(tablePartition));
     builder.addStatement(table.setOffset(kafkaPartition, offset));
 
+    // TODO(sophie): clean up this hack, perhaps by combining the offset and stream-time into
```

Comment:
I also intend to do this TODO right away as well, though again, probably in a followup PR